ExampleMod/Content/NPCs/ExamplePerson.cs

```diff
@@ -138,6 +139,30 @@ public class ExamplePerson : ModNPC
 			return score >= ((right - left) * (bottom - top)) / 2;
 		}
 
+		//Set Example Person's biome and neighbor preferences with the NPCHappiness hook. You can add happiness text and remarks with localization (See an example in ExampleMod/Localization/en-US.lang).
+		public override void NPCHappiness(int primaryPlayerBiome, ref ShopHelper shopHelperInstance, ref bool[] nearbyNPCsByType) {
+			//TODO: Make it prefer ExampleBiome when that happens. And BiomeID.
```

Comment:
This TODO should be complete as #1205  has-been merged and includes all the stuff to setup for this.
Should be a simple todo to do